lib/utils/__mocks__/github.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+//we dont want to actually hit our github stuff, that can mess up the testing...right?
```

Comment:
yep! we don't have a good way to go through the _real_ oAuth process in the tests, so we do our best by mocking the process